specification/managementgroups/ManagementGroups.Management/ManagementGroup.tsp

```diff
@@ -0,0 +1,190 @@
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-azure-resource-manager";
+import "@typespec/rest";
+import "./models.tsp";
+
+using TypeSpec.Rest;
+using Azure.ResourceManager;
+using TypeSpec.Http;
+
+namespace Microsoft.Management;
+/**
+ * The management group details.
+ */
+model ManagementGroup
+  is Azure.ResourceManager.ProxyResource<ManagementGroupProperties> {
+  ...ResourceNameParameter<
+    Resource = ManagementGroup,
+    KeyName = "groupId",
+    SegmentName = "managementGroups",
+    NamePattern = ""
+  >;
+}
+
+@armResourceOperations
+interface ManagementGroups {
+  /**
+   * Get the details of the management group.
+   *
+   */
+  get is Extension.Read<
+    Extension.Tenant,
+    ManagementGroup,
+    Parameters = {
```

Comment:
I think by definition management group should be defined as a tenant resource instead of extension - usually extension resource should use `ExtensionResource` as base type
```suggestion
@tenantResource
model ManagementGroup
  is Azure.ResourceManager.ProxyResource<ManagementGroupProperties> {
  ...ResourceNameParameter<
    Resource = ManagementGroup,
    KeyName = "groupId",
    SegmentName = "managementGroups",
    NamePattern = ""
  >;
}

@armResourceOperations
interface ManagementGroups {
  /**
   * Get the details of the management group.
   *
   */
  get is ArmResourceRead<
    Extension.Tenant,
    ManagementGroup,
    Parameters = {
```